packages/samples/angular/package.json

```diff
@@ -4,8 +4,7 @@
   "scripts": {
     "ng": "ng",
     "dev": "npm run build:assets && npm run dev:angular",
-    "dev:angular": "ng serve",
-    "link:angular": "node scripts/link-hoisted-dependencies.mjs",
+    "dev:angular": "ng serve --host=0.0.0.0",
```

Comment:
Angular's e2e test was timing out with an error that implied it didn't finish building within 90 seconds, despite it having actually finished within 30 seconds. Re-running it wouldn't fix the issue.

I saw a post that had a similar error for a seemingly unrelated issue, and their solution was to add `--host=0.0.0.0`. I was trying a lot of stuff in parallel, so I gave it a shot, and it worked. I'm still not quite sure what the root of the issue is, but if you guys think it's worth investigating, I can look into it.